xcodeproj/internal/xcodeproj_rule.bzl

```diff
@@ -345,23 +345,30 @@ def _process_targets(
         for id, label in replacement_labels.items()
     }
 
-    labels = {
+    target_labels = {
```

Comment:
What do you think about prefixing these two with `xcode_` to be abundantly clear compared to `label_strs`?